### PR TITLE
Drop <9.9

### PR DESCRIPTION
**Pull request by ganncamp**

### CODE REVIEW

**Comment by ganncamp** [comment]
@christophelevis, @agigleux I'm waiting to start the PR to end compatibility at 10.4 until after this is merged. So... it would be nice to get this reviewed and done some time soon.

File: l10nfr.properties
Comment (agigleux) [inline l10nfr.properties]: Is this going to work with the trailing comma at the end?
Can you order the versions of archivedVersions?

**Review by agigleux — CHANGES_REQUESTED** [review]

File: ansible.properties
Comment (christophelevis) [reply ansible.properties:4]: Update:
- I found, in the other plugins, a couple of versions compatible with SQ 9.9 that, I think, should be preserved in the archivedVersions list. I've added comments there.
- Also, I think we should drop the descriptions (and linksà of the versions we completely remove.

**Review by christophelevis — CHANGES_REQUESTED** [review]

File: ansible.properties
Comment (ganncamp) [reply ansible.properties:4]: I dropped the list of archived version lists because the job failed since the referenced versions of SQ weren't available. 
What are you expecting @christophelevis? That we keep version numbers in the `archivedVersions` list, but delete the entire(?) block that describes the version? Delete only the URLs from those blocks?

Comment (ganncamp) [reply ansible.properties:4]: @christophelevis can you take another look, please?

File: openedge.properties
Comment (christophelevis) [inline openedge.properties:17]: Is there a reason for removing the links for 2.24.2 and archiving it?

**Review by christophelevis — APPROVED** [review]
Thanks @ganncamp. 👍  Looks great to me.
There's just a change in openedge.properties that I don't understand.

File: openedge.properties
Comment (ganncamp) [reply openedge.properties:17]: Catching this up with another change already in Master. The maintainer removed the binary.

**Pull request merged**
